requirements.txt

```diff
@@ -1 +1,2 @@
 requests==2.27.1
+Faker
```

Comment:
Do we need specify a version of faker library?

Reply:
I think it would be better to fix the version for the stability of the framework. I will mark the current version 13.13.0.